content/copilot/configuring-github-copilot/configuring-github-copilot-in-your-environment.md

```diff
@@ -233,8 +233,8 @@ If you don't want to use the default keyboard shortcuts in {% data variables.pro
 
 If you use ReSharper, {% data variables.product.prodname_copilot %} may work best when you configure ReSharper to use {% data variables.product.prodname_copilot %}'s native IntelliSense. For more information about ReSharper, see the [ReSharper documentation](https://www.jetbrains.com/resharper/documentation/documentation.html)
 
-1. In the {% data variables.product.prodname_vs %} menu bar, under **Tools**, click **Options**.
-   ![Screenshot of {% data variables.product.prodname_vs %} menu bar. In the "Tools" menu, the "Options" item is highlighted with an orange outline.](/assets/images/help/copilot/vs-toolbar-options.png)
+1. In the Resharper menu bar, under **Extensions**, then **Resharper**, click **Options**.
+   ![Screenshot of Resharper menu bar. In the "Resharper" menu, the "Options" item is highlighted with an orange outline.](/assets/images/help/copilot/resharper-toolbar-options.png)
```

Comment:
The screenshot doesn't match the style of our screenshots. However, I don't think we need a screenshot here. We should just delete this line.

```suggestion
```